app/lib/frontend/handlers.dart

```diff
@@ -101,6 +101,7 @@ const _handlers = const <String, shelf.Handler>{
   '/web/packages': webPackagesHandlerHtml,
   '/debug': debugHandler,
   '/feed.atom': atomFeedHandler,
+  '/sitemap.txt': _siteMapHandler,
```

Comment:
Why not an xml sitemap? IIRC that is more widely adopted by search engines other than Google.